main.lua

```diff
@@ -1366,3 +1350,71 @@ function userdataIterator(data)
         if success then return idx, item end
     end
 end
+
+local LSPCmds = {
+    ["start"] = startServer,
+    ["stop"] = stopServers,
+    ["showlog"] = showLog,
+    ["sync-document"] = function (bp) syncFullDocument(bp.Buf) end,
+    ["hover"] = hoverAction,
+    ["format"] = formatAction,
+    ["autocomplete"] = completionAction,
+    ["goto-definition"] = gotoAction("definition"),
+    ["goto-declaration"] = gotoAction("declaration"),
+    ["goto-typedefinition"] = gotoAction("typeDefinition"),
+    ["goto-implementation"] = gotoAction("implementation"),
+    ["find-references"] = findReferencesAction,
+    ["document-symbols"] = documentSymbolsAction,
+    ["diagnostic-info"] = openDiagnosticBufferAction,
+}
+
+local function lspCompleter(buf)
+    local opts = {}
+
+    --Do NOT autocomplete after first argument
```

Comment:
Actually it would be great to have completions for the predefined servers after `lsp start` (although not necessarily for this PR, it can be done at a later date).